raddb/sites-available/dhcpv6

```diff
@@ -319,8 +311,34 @@ send Reply {
 	#
 	#  All replies MUST include a Server-ID option
 	#
+	&reply.Transaction-ID = &request.Transaction-ID
+
 	&reply.Server-ID.DUID = UUID
 	&reply.Server-ID.DUID.UUID.Value := 0x00000000000000000000000000000000
+
+	switch &request.Packet-Type {
+		case Request {
+			&reply += {
+				#&SOL-Max-RT = 60
+				#&DNS-Servers = fd85:d2bb:92c::1
+				#&DNS-Servers = fd85:d2bb:92c::2
+				#&Domain-List = "example.com"
+			}
+		}
+
+		case Release {
+			&reply += {
+				#&Client-ID.DUID = UUID
+				#&Client-ID.DUID.UUID.Value = 0x000102030405060708090a0b0c0d0e0f
```

Comment:
... why would we add a static DUID to the example config?  All the client ID copying gets handled automatically by the DHCPv6 process module.

Reply:
well, the current code is not doing that automatically.

Reply:
- https://github.com/FreeRADIUS/freeradius-server/blob/master/src/process/dhcpv6/base.c#L351
- https://github.com/FreeRADIUS/freeradius-server/blob/master/src/process/dhcpv6/base.c#L410
- https://github.com/FreeRADIUS/freeradius-server/blob/master/src/process/dhcpv6/base.c#L436
- https://github.com/FreeRADIUS/freeradius-server/blob/master/src/process/dhcpv6/base.c#L486
- https://github.com/FreeRADIUS/freeradius-server/blob/master/src/process/dhcpv6/base.c#L612

Feel free to submit a test case which shows that it's not doing this automatically, and we can figure out a fix.  It's probably an issue with the state table. https://github.com/FreeRADIUS/freeradius-server/blob/master/src/process/dhcpv6/base.c#L751

Reply:
...but using a switch statement like this is just wrong.  There's send sections for the various reply types, if you did want to add canned responses you'd add them there.

Reply:
This patch is going into a `send Reply { ... }` section.  The issue is that a `Reply` can be sent in response to a `Confirm`, `Solicit`, `Renew`, `Rebind` ... section.  See https://datatracker.ietf.org/doc/html/rfc8415, and look for `/Reply`.

So we may need to change what's in the reply based on the type of the request.

Perhaps the simple way is to just document this, and in the `recv Confirm` etc. sections, put the "update reply" examples there.  And then also document `send Reply` that is is processed for multiple kinds of request packets.  And therefore the `send Reply` should only contain things which are relevant for all replies.

Reply:
Yeah that works...

Reply:
Or add `send reply.confirm` or `send reply_to_confirm` sections...